pkg/project/resource_project.go

```diff
@@ -217,9 +217,18 @@ func projectResource() *schema.Resource {
 		},
 
 		"repos": {
-			Type:        schema.TypeSet,
-			Optional:    true,
-			Elem:        &schema.Schema{Type: schema.TypeString},
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+			MinItems: 0,
+			MaxItems: func() int {
+				if isOverride := getBoolEnvVar("REPO_LIMIT_OVERRIDE", false); isOverride {
+					return 1<<(32-1) - 1
```

Comment:
This is not readable code:
1. this is not obvious what the return value is
2. why is this necessary? A simple integer suffice, no?

Reply:
True. Just mimic the equivalent of Max Int. I can set to a high value like 9999 instead.

Reply:
changed to 2147483647